src/util/yarn-or-npm.js

```diff
@@ -1,4 +1,4 @@
-import { spawnPromise } from 'spawn-rx';
```

Comment:
Another spawning util?

Reply:
`cross-spawn` itself is a dep of `yarn-or-npm`, so it's technically already pulled in (aside from the Promise wrapper module). Unfortunately `spawn-rx` doesn't handle escaping metacharacters like `^`. Perhaps if `spawn-rx` was a wrapper around `cross-spawn`...

Reply:
I had a hell of a time trying and failing to get the escaping to work with my own code this evening.

Reply:
I'm considering seeing what happens if I replace all instances of `spawn-rx` and `spawn` with `cross-spawn-promise`...